packages/create-contentful-app/template/bin/create-app-definition.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+const fetch = require('node-fetch');
+const { createClient } = require('contentful-management');
+const inquirer = require('inquirer');
+const { managementToken } = require('../.contentfulrc.json');
+
+const client = createClient({
+  accessToken: managementToken
+});
+
+async function fetchOrganizations() {
+  try {
+    orgs = await client.getOrganizations();
```

Comment:
this is a linting error in the output, as orgs is never initialised. 


```suggestion
    const orgs = await client.getOrganizations();
```